test/System/Err.cpp

```diff
@@ -0,0 +1,38 @@
+#include <SFML/System/Err.hpp>
+#include <sstream>
+
+#include <doctest.h>
+
+TEST_CASE("sf::err - [system]")
+{
+    SUBCASE("Overflow default buffer")
+    {
+        sf::err() << "SFML is a simple, fast, cross-platform and object-oriented multimedia API."
+            "It provides access to windowing, graphics, audio and network."
+            "It is written in C++, and has bindings for various languages such as C, .Net, Ruby, Python.";
+    }
+
+    SUBCASE("Redirect buffer to observe contents")
+    {
+        sf::err() << "We'll never be able to observe this" << std::endl;
```

Comment:
Maybe comment that the `endl` is truly required for the flush, not that this gets replaced by `\n` in the future.

Reply:
Done